src/components/modals/ActionCreatorEditor.tsx

```diff
@@ -979,8 +979,8 @@ class ActionCreatorEditor extends React.Component<Props, ComponentState> {
                                 </div>
                                 {!this.state.isPayloadValid &&
                                     (<div>
-                                        <p className="ms-TextField-errorMessage css-83 errorMessage_20d9206e">
```

Comment:
Appear to both have been a copy paste error.  "errorMessage_20d9206e" is also not a valid style that could potentially introduce bad styling and should be removed